api/DOMTokenList.json

```diff
@@ -429,6 +477,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "remove_multiple_arguments": {
+          "__compat": {
+            "description": "Multiple arguments for <code>remove()</code>",
+            "support": {
+              "chrome": {
+                "version_added": "24"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "26"
+              },
+              "firefox_android": {
+                "version_added": "26"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "15"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "7"
+              },
+              "safari_ios": {
+                "version_added": "7"
+              },
+              "samsunginternet_android": {
+                "version_added": true
+              },
+              "webview_android": {
+                "version_added": true
```

Comment:
≤37

Reply:
I've added another commit with your corrections, but it should be noted that I just copied those from the classList compat info, so I assume it would have to be changed there too?

Reply:
That would be great if we can update them there as well, yeah!  Since they’re set to “true”, we’ll probably run the upcoming data mirroring script to fill in those version numbers, but since we’re thinking about it we might as well perform the updates manually.